### DISCUSSION TITLE
TypeError: Cannot read property 'queries' of undefined

### DISCUSSION THREAD

**Discussion by ABoyd002** [strapi/strapi#6043] [Legacy Topics]
Hello, I am having an issue when I try to run `npm run develop` I keep getting this error,

```
$ strapi develop
[2020-05-03T18:35:32.297Z] debug ⛔️ Server wasn't able to start properly.
[2020-05-03T18:35:32.299Z] error TypeError: Cannot read property 'queries' of undefined
ger.js:91:7)
    at createWebhookStore (C:\Users\arran\WebstormProjects\wynx\wynx_backend\node_modules\strapi\lib\services\webhook-store.js
:57:29)
    at Strapi.load (C:\Users\arran\WebstormProjects\wynx\wynx_backend\node_modules\strapi\lib\Strapi.js:278:25)
    at async Strapi.start (C:\Users\arran\WebstormProjects\wynx\wynx_backend\node_modules\strapi\lib\Strapi.js:130:7)
error Command failed with exit code 1.
```
I was using v3.0.0-beta.20 when this happend and tried to update the packages but I still get the error.

**System**

- Node.js version: 12.16.3
- NPM version: 6.13.6
- Strapi version: v3.0.0-beta.20.1
- Database: SQLite
- Operating system: Windows 10

**Comment by derrickmehaffy** [comment]
@ABoyd002 this is likely due some cached package, purge the following files/folders and reinstall/rebuild

- `build`
- `node_modules`
- `.cache`
- `package-lock.json`
- `yarn.lock`

I'm going to close this as it isn't an issue with Strapi directly but if you need further help don't hesitate to reply

**Comment by ABoyd002** [comment]
@derrickmehaffy I deleted all the files/folders you said to but I still get the same error

**Comment by derrickmehaffy** [comment]
Do you have any custom controllers/services or any modifications to plugins via extensions?

**Reply by derrickmehaffy** [reply]
I've moved this over to a discussion also

**Reply by ABoyd002** [reply]
I have one custom controller that sends an email with send in blue

**Reply by derrickmehaffy** [reply]
Is your project code somewhere public like on github you can link to? Right now it's basically impossible to properly debug whats going on.

**Reply by ABoyd002** [reply]
https://github.com/ABoyd002/wynx_backend

**Reply by derrickmehaffy** [reply]
Two things:

- `"strapi": "^3.0.0-next.55",` this is going to cause problems, you shouldn't be using next versions
- `"strapi-provider-email-sendinblue": "^1.0.5",` This could also cause problems, you may want to check when this was updated as it may not be entirely up to date with the latest changes (contact the author)

**Reply by derrickmehaffy** [reply]
Also

https://github.com/ABoyd002/wynx_backend/blob/74b5a6cfa061bf743dff5dbfc55b7f9831c6e488/api/messages/controllers/messages.js#L55

Is there a reason you are using strapi.query here? instead of just passing the `messages` string?

**Reply by ABoyd002** [reply]
Thanks for your help, I now have it working.
